component/class/defaults.yml

```diff
@@ -10,3 +9,11 @@ parameters:
         registry: 'ghcr.io'
         repository: 'vshn/cloudscale-metrics-collector'
         tag: 'v0.2.0'
+  appuio_cloud_reporting:
+    namespace: 'appuio-cloud-reporting'
+    database:
+      name: 'reporting'
+      host: 'reporting-db.appuio-reporting.svc'
+      parameters: 'sslmode=disable'
+      password: 'passw0rd'
+      port: 5432
```

Comment:
Maybe this is better defined in `component/tests/defaults.yml`? Since they are only relevant during local compilation.

Reply:
Makes sense, done.